docs/references/files/fossa-yml.md

```diff
@@ -26,6 +26,24 @@ project:
     - project-label-1
     - test-project
 
+releaseGroup:
+  title: release-group-title
+  release: release-group-release
+  releaseGroupProjects:
+    - projectLocator: custom+123/git@github.com/fossas/fossa-cli
+      projectRevision: "12345"
+      projectBranch: master 
+    - projectLocator: custom+123/git@github.com/example
```

Comment:
We discussed project id vs. locator with respect to modifying projects and I think we should do the same here. This is technically an entirely new config block so there's no compatibility issues but users are already used to using `projectId` elsewhere. 

I think it'd be good to allow/prefer `projectId` and make `projectLocator` available for people who really need it.

Reply:
I think allowing projectId makes sense when modifying projects as that field was already present in the `project` config block. However, in this case, because we are introducing a new config block I would rather constrain users to using `projectLocator` and make it simple. I think if we get requests to include `projectId` as an option, then we can go ahead and add it, but for now I would like to keep it as is. What do you think?

Reply:
Sounds good I updated the docs!